### DISCUSSION TITLE
Is the Lively.SCR screensaver file safe.

### DISCUSSION THREAD

**Discussion by BraydenGamelover** [rocksdanister/lively#1504] [Q&A] [answered]
I have heard some .SCR files can be viruses. I also cannot download the file even if I disable my antivirus.

**Comment by rocksdanister** [comment, marked as answer]
.SCR files are just renamed .EXE applications, so yes some files can do harm.

However Lively .SCR file is safe.. all its doing is sending a message to Lively, **source code**:
https://github.com/rocksdanister/lively/blob/core-separation/src/Lively/Lively.Screensaver.Minimal/Program.cs
You can **compile** and check or use alternative screensaver install methods listed in the wiki.

Some **Anti-Virus** program heuristic algorithms may falsely flag Lively files (I only verify with built in Microsoft Security.)
You can help out by reporting **False-Positives** directly to the software vendor:
https://www.autohotkey.com/boards/viewtopic.php?t=62266
